src/models.py

```diff
@@ -41,3 +48,33 @@ def _is_valid(address: str) -> bool:
             flags=re.IGNORECASE
         )
         return match_result is not None
+
+
+@dataclass
+class InternalTokenTransfer:
+    """Total amount reimbursed for accounting period"""
+    transfer_type: str
+    from_token_address: Address
+    to_token_address: Optional[Address]
+    from_amount: int
+    to_amount: Optional[int]
```

Comment:
They are the same in terms of value, but they are different in terms of token amounts

Reply:
I'm sorry, but I don't know what this means. Why is one optional and the other not?

Reply:
I guess what you are looking for is nested struct. Definitely can use enum a too. Anyway, I still don't know what these optional fields are meant to be for and it seems we don't need them so it will be nice to remove them.